UnityProject/Assets/Scripts/Player/PlayerSync.Server.cs

```diff
@@ -395,18 +400,29 @@ private PlayerState NextStateServer(PlayerState state, PlayerAction action)
 		}
 	}
 
+	/// <summary>
+	/// Attempts to push things or open doors
+	/// </summary>
+	/// <param name="currentPosition">current world position</param>
+	/// <param name="direction">direction of movement</param>
 	private void BumpInteract(Vector3 currentPosition, Vector3 direction) {
 			StartCoroutine( TryInteract( currentPosition, direction ) );
 	}
 
+	/// <summary>
+	/// Tries to interact in a direciton, trying to open a closed door and push something.
+	/// </summary>
+	/// <param name="currentPosition">current world position</param>
+	/// <param name="direction">direction of movement</param>
+	/// <returns></returns>
 	private IEnumerator TryInteract( Vector3 currentPosition, Vector3 direction ) {
 		var worldPos = Vector3Int.RoundToInt(currentPosition);
 		var worldTarget = Vector3Int.RoundToInt(currentPosition + direction);
 
 		InteractDoor(worldPos, worldTarget);
 
 //		Logger.LogTraceFormat( "{0} Interacting {1}->{2}, server={3}", Category.Movement, Time.unscaledTime*1000, worldPos, worldTarget, isServer );
-		InteractPushable( worldTarget, direction );
+		InteractPushable(worldPos, direction );
```

Comment:
That solved a bug I was encountering when pushing. I can add logic to ensure the previous behavior works.

Reply:
NVM, this will be fixed as part of a different ticket #1110 so it would be throwaway work to fix it here since the way door closing works will be different